cliquet/events.py

```diff
@@ -41,25 +41,47 @@ def __init__(self, action, timestamp, read_records, request):
 
 
 class ResourceChanged(_ResourceEvent):
-    """Triggered when a resource is changed.
+    """Triggered when a resource is being changed.
```

Comment:
We might want to add a note here about why this resource is useful, and that it doesn't guarantee that it hasn't been rolledback.